src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,166 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I was looking for other examples and found Microsoft lib. It has the same styling for anchor and button breadcrumbs. Maybe we should do the same? @FlyersPh9 https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb

Off topic, but I really like their demo pages with a short description and best practices for the component.

Reply:
Personally I don't mind the different visuals, but we could style the anchor link within a breadcrumb to look and interact more like a button.

Reply:
After some discussion, we've decided to keep the different visuals but were able to remove the `BreadcrumbItem` component (by improving the CSS). The user can now pass any kind of element and we'll handle most cases properly.

And I've saved that FluentUI documentation link in #226.